.github/CODEOWNERS

```diff
@@ -1 +1 @@
-.github/CODEOWNERS @sonarsource/sonarqube-team
+.github/CODEOWNERS @sonarsource/devex-devandteamworkflow-squad
```

Comment:
I thought this repository was under the analysis-experience Squad's responsibility.